powerhome/playbook
# [PLAY-70]Nav Kit new "bold" variant

#### Screens

(Reference given for inspiration)
<img width="1386" alt="image" src="https://user-images.githubusercontent.com/84349455/150378273-48e840a2-f4b3-4bf8-b924-e1c713a8fde7.png">

(Vertical Example)
<img width="1465" alt="image" src="https://user-images.githubusercontent.com/84349455/150378535-a44e7208-b144-45bc-836e-6cf8be5114da.png">

(Horizontal Example)
<img width="1482" alt="image" src="https://user-images.githubusercontent.com/84349455/150378581-af296683-8eb6-4814-bc71-1735772739f2.png">

### NOTES: 
- This PR adds Rails and React Doc examples for the bold variant, showcasing it being used on both horizontal and vertical navs.
- `background-color: darken($primary, 4%)` is applied to an active nav item on hover to mimic the treatment of hover on primary button kits.
- Using icons was not part of the AC, but the `bold` variant works with icons on both sides of the nav item.


#### Breaking Changes

No, this PR adds a new `bold` value to the "variant" prop in the Nav Kit with associated styling, so the existing React/Rails logic and scss files for "pb_nav" are unchanged.


#### Runway Ticket URL

[https://nitro.powerhrg.com/runway/backlog_items/PLAY-70](url)

#### How to test this

1. Use "Nav" kit in an .html.erb or .jsx file with a vertical or horizontal orientation
2. Pass "bold" to the variant prop
3. Check to make sure "active" nav items reflect the screenshots above

#### Checklist:
